sql/createTables.sql

```diff
@@ -19,7 +20,7 @@ CREATE TABLE Location (
 ) ENGINE = InnoDB;
 
 CREATE TABLE Space (
-    SpaceID INT,
+    SpaceID INT AUTO_INCREMENT,
```

Comment:
This field doesn't need to be auto_increment

Reply:
changed